src/components/Film/index.js

```diff
@@ -69,32 +46,7 @@ function Film(){
                         </button>
                     </footer>      
                 </div>  
->>>>>>> main
             </div>
-            {/*We should sent films.movie_watched */}
-            <FilmWatched filmWatched={false} />
-          </div>
-          <div className="film-image">
-            <img src={film.movie_banner} alt="" />
-          </div>
-          <StarRating />
```

Comment:
We have differences. Because in this line we should instance the new component DoubleRating. This change I have it committed.